man/less-variables.xml

```diff
@@ -94,9 +94,9 @@
     <varlistentry id='colors'>
       <term><varname>$SYSTEMD_COLORS</varname></term>
 
-      <listitem><para>The value must be a boolean. Controls whether colorized output should be
-      generated. This can be specified to override the decision that <command>systemd</command> makes based
-      on <varname>$TERM</varname> and what the console is connected to.</para>
+      <listitem><para>The value can be either <literal>0</literal>, <literal>16</literal> or <literal>256</literal>.
```

Comment:
Strictly speaking, this would break backwards compatiblity by not allowing SYSTEMD_COLORS=1 or SYSTEMD_COLORS=true and such. This needs to be allowed still.

Reply:
Uhm, this will complicate a bit the parsing. I can't simply use getenv_int anymore.